enums/foodTypeEnum.js

```diff
@@ -0,0 +1,35 @@
+const foodTypeEnum = {
+  RAW_MEAT: {
```

Comment:
I think this key/value has changed at their request - full description for value and amended key value needed. Make sure updated in front-end if required